src/main/api/2/studio-api-2.yaml

```diff
@@ -3287,6 +3397,299 @@ paths:
+      operationId: getTranslationItem
+      parameters:
+        - name: siteId
+          in: query
+          description: site ID
+          required: true
+          schema:
+            type: string
+        - name: path
+          in: query
+          description: item path
+          required: true
+          schema:
+            type: array
+            items:
+              type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  response:
+                    $ref: '#/components/schemas/ApiResponse'
+                  item:
+                    $ref: '#/components/schemas/LocalizationItem'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+
+  /translation/list_target_locales:
+    get:
+      tags:
+        - translation
+      summary: Get list of target translation locales
+      description: 'Required permission "get_translation_target_locales"'
+      operationId: getTranslationTargetLocales
+      parameters:
+        - name: siteId
+          in: query
+          description: site ID
+          required: true
+          schema:
+            type: string
+        - name: path
+          in: query
+          description: item path
+          required: true
+          schema:
+            type: array
+            items:
+              type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  response:
+                    $ref: '#/components/schemas/ApiResponse'
+                  items:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/LocalizationItem'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+
+  /translation/mark_for_translation:
+    post:
+      tags:
+        - translation
+      summary: Request item to be translated
+      description: 'Required permission "mark_for_translation"'
+      operationId: markForTranslation
+      requestBody:
+        description: Item and list of locales to translate to
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                siteId:
+                  type: string
+                  description: site ID
+                path:
```

Comment:
Can we support a list of items, not just one?
Can this also support a list of item IDs not only a list of paths?

Reply:
What about a list of IDs?